ggml/src/ggml-cpu/ggml-cpu-quants.c

```diff
@@ -5573,88 +5986,7 @@ void ggml_vec_dot_q4_K_q8_K(int n, float * restrict s, size_t bs, const void * r
 
     uint32_t utmp[4];
 
-#ifdef __ARM_FEATURE_SVE
```

Comment:
This was probably removed by mistake.

Reply:
Yup I fixed in the latest version, should be because I copy this file from an old version of ggml